gulpfile.babel.js

```diff
@@ -288,6 +288,18 @@ export const keycheck = () => (
       })
 );
 
+// eslint-disable-next-line camelcase
+export const bitly_keycheck = () => (
```

Comment:
Doesn't matter, but why is this wrapped in parens? Isn't it just a single expression?

Reply:
I think this might be an eslint thing? I'll revert and see if it switches back after an eslint pass

Reply:
Doesn't seem like an eslint thing. I think I just did it for consistency with the `{}` arrow functions.